src/poprox_recommender/domain/account.py

```diff
@@ -0,0 +1,8 @@
+from uuid import UUID
+
+from pydantic import BaseModel
+
+
+class Account(BaseModel):
+    account_id: UUID = None
+    email: str
```

Comment:
This isn't actually used in the recommender code but is used in the associated platform code in `poprox-core`. I'm putting it here anyway so that all the domain object models are in the same package, which will make it easier to extract them from these repos into a separate repo/package in the future